### PR TITLE
add egi account documentation

### PR DESCRIPTION
**Pull request by dthiex**
Closes #210

### CODE REVIEW

File: join/early_adopter.md [hunk 1]
```diff
@@ -143,6 +143,10 @@ the platform through any of the clients. With all clients you will need to conne
 `https://openeo.cloud` and then authenticate through EGI check-in with the 
 account used above.
 
+::: warning Recommendation
+We recommend that you always use the same identity provider that you originally signed up with when identifying yourself through EGI Check-in. This is the only account that is authorized by and has access to openEO Platfrom. However, it is possible to link several identiy providers with your EGI account. Read more about this in the EGI documentation [here](https://docs.egi.eu/users/aai/check-in/linking/).
```
Comment (soxofaan) [inline join/early_adopter.md:147]: > This is the only account that is authorized by and has access to openEO Platfrom.

I don't think that is correct: I have a second provider linked to my EGI account, and things seem to work fine regardless of which provider I use to I log in.

Comment (jdries) [reply join/early_adopter.md:147]: So it's more like: "If you want to use a different provider then the one you originally signed up with, you have to first link the accounts yourself."?
Nice find on the linking docs, that's the bit I was missing when I created the issue.

Comment (soxofaan) [reply join/early_adopter.md:147]: Indeed. In that sense, this "warning" paragraph is a bit too alarmist  I think. 
We can just inform that you can link multiple Id providers to the same EGI account.

I think the problem of Patrick is that he created a new EGI account (because his original one was temporarily unavailable).

Comment (dthiex) [reply join/early_adopter.md]: > I think the problem of Patrick is that he created a new EGI account (because his original one was temporarily unavailable).

Yes that's true and for the case where your original one is temporarily unavailable, you can also not link another one because you can't access it. Still I think it is good to explain that we recommend using always the same account and how to link accounts. 

It's labelled as warning but the recommendation is yellow and will also be displayed as such. @soxofaan Do you still find this to alarmist? I updated the wording to reflect better what is going on.

grafik

Comment (soxofaan) [reply join/early_adopter.md]: > Still I think it is good to explain that we recommend using always the same account 

Do you mean the same EGI account, or the same Identity Provider?
Assuming you mean the latter: I don't understand why we should recommend that. Isn't the whole point of using EGI that we (openeo Platform) don't have to care about all the identity provider details?

I understand that we want to promote using an institutional account for the initial EGI account setup, but after that, it doesn't/shouldn't really matter I think.

Comment (dthiex) [reply join/early_adopter.md]: > it doesn't/shouldn't really matter I think.

It matters because if the user switches Identity Providers and doesn't link them before to the EGI account he used for signing up to the platform he will not be able to use the platform. 

In that sense recommending using always the same identity provider will be more convenient for the user (and thus something I'd recommend) because they don't need to link any accounts.

Comment (soxofaan) [reply join/early_adopter.md]: > Do you still find this to alarmist?

I think your screenshot is from a different place on that page, but I checked the diff.

It's indeed better, but I think it can be just a `tip` block instead of ` `warning` block.

On a more general note: I think the whole document is already quite complex with a lot of boxes, side-tracks, screenshot based flows, etc. So I would refrain from making it even more complex.

Can't we amend the `::: tip You have already connected your account in the past?` block with this info instead of starting a new one?

Comment (soxofaan) [reply join/early_adopter.md]: > It matters because if the user switches Identity Providers ...

I'm probably technically biased, but isn't it common knowledge that using a different IdP implies ending up in a different account by default?

> In that sense recommending using always the same identity provider will be more convenient for the user (and thus something I'd recommend)

Ok I understand better now where you're coming from. I read the "We recommend that ..." as  admitting a shortcoming.

Comment (dthiex) [reply join/early_adopter.md]: > I'm probably technically biased, but isn't it common knowledge that using a different IdP implies ending up in a different account by default?

Personally I was suspecting that this would be the case but think for a lot of user´s this will not be clear so think it's good to explicit mention it.

> I read the "We recommend that ..." as admitting a shortcoming.

No it was just meant to hint how things will work easier for the user. Would changing recommend to advise be better?

> Can't we amend the ::: tip You have already connected your account in the past? block with this info instead of starting a new one?

I would prefer to leave it within the current section as it's connected to the log in and not the registration part (I feel the first part is only about the registration).

File: join/early_adopter.md [hunk 2]
```diff
@@ -143,6 +143,10 @@ the platform through any of the clients. With all clients you will need to conne
 `https://openeo.cloud` and then authenticate through EGI check-in with the 
 account used above.
 
+::: tip Tip
+We advise you to always use the same identity provider  you originally signed up with when identifying yourself through EGI Check-in (as this is the one associated with your EGI account authorized by the platform). If you want to use a different provider than the one you originally signed up with, you must first link the new account to your existing EGI account. Read more about how to link different identity providers to your EGI account in the EGI documentation [here](https://docs.egi.eu/users/aai/check-in/linking/).
```
Comment (soxofaan) [inline join/early_adopter.md:147]: The first sentence is a bit hard to parse as intended I think

suggestion:

> We advise you, for your own ease of use, to always log in with the same identity provider you originally registered with. Otherwise you risk ending up with a separate new EGI account, that again has to go through the openEO Platform virtual organization acceptance flow.
> It is however possible to link multiple accounts from multiple identity providers to the same EGI account, but this has to happen before you use these accounts to log in as explained in the [EGI documentation](https://docs.egi.eu/users/aai/check-in/linking/).

Comment (dthiex) [reply join/early_adopter.md]: Thanks for the suggestion. I rearranged the word order a bit but otherwise copied it.

**Review by soxofaan — APPROVED** [review]

File: join/early_adopter.md (hunk 2 above)
Comment (soxofaan) [reply join/early_adopter.md]: great!
